docs/en/siem/cases-api-create.asciidoc

```diff
@@ -0,0 +1,90 @@
+[[cases-api-create]]
+=== Create case
+
+Creates a new case.
+
+==== Request URL
+
+`POST <kibana host>/<port>/api/cases`
+
+==== Request body
+
+A JSON object with these fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`title` |String |The case's title. |Yes
+|`description` |String |The case's description. |Yes
+|`status` |String a|Indicates whether the case is open or closed. Valid values 
+are:
+
+* `open`
+* `closed`
+
+|Yes
+|`tags` |String[] |String array containing words and phrases that help
+categorize cases. |Yes, can be an empty array.
+|==============================================
+
+===== Example request
+
+[source,sh]
+--------------------------------------------------
+POST api/cases
+{
+  "description": "James Bond clicked on a highly suspicious email
+  banner advertising cheap holidays for underpaid civil servants.",
+  "title": "This case will self-destruct in 5 seconds",
+  "status": "open",
+  "tags": [
+    "phishing",
+    "social engineering"
+  ]
+}
+--------------------------------------------------
+// KIBANA
+
+==== Response code
+
+`200`:: 
+   Indicates a successful call.
+
+==== Response payload
+
+A JSON object that includes the user who created the case and the case's ID,
+version, and creation time.
+
+===== Example response
+
+[source,json]
+--------------------------------------------------
+{
+  "id": "a18b38a0-71b0-11ea-a0b2-c51ea50a58e2",
+  "version": "Wzc0LDFd",
+  "comments": [],
+  "totalComment": 0,
+  "closed_at": null,
+  "closed_by": null,
+  "created_at": "2020-03-29T11:30:02.658Z",
+  "created_by": {
+    "email": "ahunley@imf.usa.gov",
+    "full_name": "Alan Hunley",
+    "username": "ahunley"
+  },
+  "external_service": null, <1>
```

Comment:
why is there a `<1>` here?

Reply:
Hi @stephmilovic 

The '<1>' is a callout in the transformed HTML docs. Like [this](http://stack-docs_761.docs-preview.app.elstc.co/guide/en/siem/guide/master/cases-api-create.html#_example_response_3).